source/developer/mobile-developer-setup.rst

```diff
@@ -806,44 +806,34 @@ this guide will only focus about the changes needed in the **mattermost-push-pro
 Overriding assets & White labeling
 -----------------------------------
 
-Sometimes you end up wanting to white label the Mattermost app with your organization assets to have a sense of ownership. With this in mind
-we've made the mobile apps easy to white label and to override the assets used, the downside is that you have to `Build your own app from source`_.
+We've made it easy to easy to white label the mobile app and to replace override the assets used, however, you have to `Build your own app from source`_.
```

Comment:
Easy to easy?

Reply:
That reminds me of the
the illusion where words
are doubled up and you
don't notice because they're
on two separate lines like
the "the" on the first and
second lines